test/parallel/parallel.status

```diff
@@ -112,4 +112,4 @@ test-inspector-async-hook-setup-at-inspect-brk: PASS, FLAKY
 
 [$arch==loong64]
 # https://github.com/nodejs/node/issues/51662
-test-http-correct-hostname: SKIP
+http/test-correct-hostname: SKIP
```

Comment:
```suggestion
http/test-correct-hostname: SKIP

```

Reply:
I'll make these changes (and others) once this is unblocked